types/meteor/react-meteor-data.d.ts

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 
 export function withTracker<TDataProps, TOwnProps>(
     options: (props: TOwnProps) => TDataProps | { getMeteorData: (props: TOwnProps) => TDataProps; pure?: boolean },
-): (reactComponent: React.ComponentType<TOwnProps & TDataProps>) => React.ComponentClass<TOwnProps>;
```

Comment:
I'm not familiar with the `useTracker` API, but this change doesn't look right to me. If you look at https://github.com/meteor/https://github.com/meteor/react-packages/blob/cccd609c04295722bd6427d351111586069e0618/packages/react-meteor-data/withTracker.tsx#L19, it seems like it's passing through the input props to the wrapped component, not just the props that are retrieved by the tracker function.